data/transport missions.txt

```diff
@@ -1951,3 +1951,244 @@ ship "Science Drone"
+			
+			label choice
+			choice
+				`	(Follow the figure.)`
+				`	(Return to my ship.)`
+					decline
+			
+			`	You step into the alleyway and find the figure standing under a lamp. The figure removes their cloak, revealing the face of a woman with short dark hair and a distinct jagged scar across her right cheek. She begins whispering to you. "You're not from this world. I can tell. You're too clean and unscarred. Would you be able to help me?"`
+			choice
+				`	"What do you want?"`
+					goto need
+				`	"Why would I help a pirate?"`
+			
+			`	The woman chuckles. "Not everyone who lives on a pirate world is a pirate, you know. Maybe you need to learn what a stereotype is. Regardless..." she trails off.`
+			
+			label need
+			`	"It isn't what I want, it's what I need," she whispers, putting heavy emphasis on the word "need."`
+			`	"You undoubtedly know about the horrendous slave trade that still exists on pirate worlds today. A slave trade that the Republic doesn't care to abolish, leaving innocent people to waste away like rats in a cage, working for their lives. As if the politicians of the Republic aren't vermin themselves... I'm part of a group of people who have decided to take matters into our own hands. We're mostly people who grew up on pirate worlds, but some of us were born elsewhere."`
+			`	"I assasinated a pirate warlord a couple of days ago and I have been sheltering his slaves ever since. What I need from you is to transport the <bunks> people in my care to <destination>. Another member of my group should be there to pay you <payment> and take the people somewhere far away from here." She cocks her head to the right. "Will you fight for justice and transport these people, Captain?" The look in her eyes suggests that she won't take no for an answer.`
+			choice
+				`	"Anything for the freedom of the innocent."`
+				`	"Sorry, but I'm not going to risk my life for your cause."`
+					goto decline
+			
+			`	"Good choice, Captain. We'll wait till the sun has fully set, then I will bring the people to your ship. I'll be traveling in my own ship and launch ahead of you, which should hopefully distract the pirates looking for the killer of their warlord and give you enough room to escape. Wait for me after you land on <planet>."`
+				accept
+			
+			label decline
+			apply
+				set "denied freeing slaves"
+			`	"Poor choice, Captain." The woman pulls a knife from under her cloak and causes the glare of the lamp above her to blind you. You stumble backward in anticipation of an attack, but the woman does not move. "I'll remember this, Captain," she hisses at you. The woman puts her cloak back up and sprints down the alleyway into darkness.`
```

Comment:
> The woman pulls a knife from under her cloak and causes the glare of the lamp above her to blind you.

Something doesn't feel right about the middle of this. Maybe rewrite it as, `... under her cloak, causing the glare of ...`